modules/ipi-install-network-requirements.adoc

```diff
@@ -161,4 +161,4 @@ You can reconfigure the control plane nodes to act as NTP servers on disconnecte
 [id='network-requirements-out-of-band_{context}']
 == Port access for the out-of-band management IP address
 
-The out-of-band management IP address is on a separate network from the node. To ensure that the out-of-band management can communicate with the provisioner during installation, the out-of-band management IP address must be granted access to port `80` on the bootstrap host and port `6180` on the {product-title} control plane hosts.
+The out-of-band management IP address is on a separate network from the node. To ensure that the out-of-band management can communicate with the provisioner during installation, the out-of-band management IP address must be granted access to port `80` on the bootstrap host and port `6180` on the {product-title} control plane hosts. TLS port `6183` is also required for bare metal installation via redfish.
```

Comment:
I think it's better to say "via virtual media (e.g. Redfish)" because there are other virtual media drivers

Reply:
Beautiful, thank you for that feedback, @dtantsur I'll make that change.